ceph.spec.in

```diff
@@ -493,10 +493,12 @@ Requires:       python%{python3_pkgversion}-six
 %if 0%{?fedora} || 0%{?rhel}
 Requires:       python%{python3_pkgversion}-cherrypy
 Requires:       python%{python3_pkgversion}-werkzeug
+Requires:       python%{python3_pkgversion}-yaml
```

Comment:
could you conditionalize like
```
%if 0%{?fedora} || 0%{?rhel}
....
%if 0%{?rhel} == 7
Requires:       python%{python3_version_nodots}-PyYAML
%else
Requires:       python%{python3_version_nodots}-pyyaml
%endif
```
?

Reply:
it's not ``orchestrator``, and always-on.. so it needs to go in this package.

Reply:
`orchestrator` is an arch independent python module, that's why i think we should package in a separated package.

Reply:
That's true for all mgr modules. If we want to have an arch independent package, we should package all mgr modules (except for rook,dashboard) in this new package.

Reply:
see #33422